nestkernel/conn_builder.cpp

```diff
@@ -351,19 +343,19 @@ nest::ConnBuilder::single_connect_( index snode_id, Node& target, thread target_
 
   for ( size_t indx = 0; indx < synapse_model_id_.size(); ++indx )
```

Comment:
I am confused by `synapse_model_id_` (and the little informative `indx`). This loop iterates over the different synapse specifications provided in a collocated synapse object, doesn't it? Can we find better names to make this clearer?